README.md

```diff
@@ -35,6 +35,10 @@ Note: This composite resource requires the **Package** resource that ships with
     - xWebDeploy
     - xWebPackageDeploy
 
+### Unreleased
+
+-  Added support for passing $WebDeployMsi parameter in xWebDeploy.Schema.psm1.
```

Comment:
I would like to see also the reason for this change: 'fix problems with setup on machine with no internet access'.